scripts/utils/fee_calculation_functions.py

```diff
@@ -0,0 +1,103 @@
+from utils.data_model.chain_json_model import Chain
+from utils.useful_functions import create_connection_by_url, deep_search_in_object
+
+WEIGHT_PER_SECOND = 1_000_000_000_000
+WEIGHT_PER_MILLIS = WEIGHT_PER_SECOND / 1000  # 1_000_000_000
+WEIGHT_PER_MICROS = WEIGHT_PER_MILLIS / 1000  # 1_000_000
+WEIGHT_PER_NANOS = WEIGHT_PER_MICROS / 1000  # 1_000
+
+
+def base_fee_per_second(ExtrinsicBaseWeight):
+    base_weight = ((ExtrinsicBaseWeight / WEIGHT_PER_NANOS)
```

Comment:
Why not just write ```ExtrinsicBaseWeight``` because of WEIGHT_PER_SECOND / (WEIGHT_PER_NANOS * WEIGHT_PER_MILLIS) = 1?

Reply:
I've changed that place, thank you!